netbout-web/src/main/java/com/netbout/rest/login/TkRegister.java

```diff
@@ -59,11 +59,17 @@
         this.base = bse;
     }
 
-    // @todo #602:30min/DEV During the user registration if the
-    //  `invite-key` param presented in the original ticket (destination
-    //  url) it should decrypt that value and use it to search for alias.
-    //  Using that alias, you must update urn and alias to use the alias
-    //  that was choosen by the user.
+    /**
+     * Register.
+     * @param req Request
+     * @return Response
+     * @throws IOException if fails
+     * @todo #602:30min/Dev During the user registraton if the
```

Comment:
@mstzn36 `@todo #751:30min/Dev During the user registraton if the`
You should use your ticket number here